From: SeongJae Park <sj@kernel.org>
Subject: mm/zswap: mark zswap_stored_incompressible_pages as static
Date: Thu, 21 Aug 2025 09:10:57 -0700

Only zswap.c uses zswap_stored_incompressible_pages, but it is not marked
as static.  This incurs a sparse warning that reported by kernel teset
robot.  Mark it as a static variable to eliminate the warning.

Link: https://lkml.kernel.org/r/20250821161750.78192-1-sj@kernel.org
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202508211706.DnJPQQMn-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/zswap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/zswap.c~mm-zswap-store-page_size-compression-failed-page-as-is-fix
+++ a/mm/zswap.c
@@ -45,7 +45,7 @@
 /* The number of pages currently stored in zswap */
 atomic_long_t zswap_stored_pages = ATOMIC_LONG_INIT(0);
 /* The number of incompressible pages currently stored in zswap */
-atomic_long_t zswap_stored_incompressible_pages = ATOMIC_LONG_INIT(0);
+static atomic_long_t zswap_stored_incompressible_pages = ATOMIC_LONG_INIT(0);
 
 /*
  * The statistics below are not protected from concurrent access for
_
